platform/kube/conversion_test.go

```diff
@@ -148,7 +148,7 @@ func TestServiceConversion(t *testing.T) {
 			len(service.Ports), len(localSvc.Spec.Ports))
 	}
 
-	if service.ExternalName != "" {
+	if service.ExternalName != "" || service.External() {
```

Comment:
Should just be `service.External()`, since
```go
// External predicate checks whether the service is external
func (s *Service) External() bool {
	return s.ExternalName != ""
}
```